.github/workflows/publish-container-images.yml

```diff
@@ -13,18 +13,63 @@ on:
 # tag the container image
 #
 #on:
+#  workflow_dispatch:
 #  push:
 #    branches:
 #      - master
 #      - development
 
+env:
+  IMAGE_TAG: ${{ github.ref_name }}
+  REGISTRY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+
 jobs:
   publish-ops-server:
     runs-on: ubuntu-latest
     defaults:
       run:
         shell: bash
         working-directory: ops-server
+    env:
+      IMAGE_NAME: infinite-industries/ops-server
```

Comment:
I don't quite understand what ops-server is?

Reply:
A previous PR created ops-server: this is the replacement DB backup process.